RELEASE_NOTES.md

```diff
@@ -6,9 +6,21 @@ Chef 15 release notes will be added here as development progresses.
 
 ## Breaking Changes
 
+### Omnibus chef packages now removes /opt/chef before installation
```

Comment:
Just call them Chef packages. We've mostly scrubbed the word omnibus from our various docs pages